source/routing/raptor_api.cpp

```diff
@@ -117,7 +117,25 @@ static void fill_section(pbnavitia::Section *pb_section, const type::VehicleJour
         add_coord(stop_times.front()->journey_pattern_point->stop_point->coord, pb_section);
         add_coord(stop_times.back()->journey_pattern_point->stop_point->coord, pb_section);
     }
-    fill_co2_emission(pb_section, vj);
+    fill_co2_emission(pb_section, d, vj);
+}
+
+void co2_emission_aggregator(pbnavitia::Journey* pb_journey){
```

Comment:
as this method is only used in this file, and not exported in the .h, please add "static" before:
```c++
static void co2_emission_aggregator(pbnavitia::Journey* pb_journey){
```